modules/API/nav.adoc

```diff
@@ -1,11 +1,3 @@
 * xref:index.adoc[REST API Reference]
-<<<<<<< HEAD
-** xref:authentication.adoc[]
-** xref:built-in-endpoints.adoc[Built-in Endpoints]
```

Comment:
we should add this back, we may rename it to `RESTPP & Infra Endpoints` since most of them are actually RESTPP APIs, and few of them are Infra APIs